packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -240,7 +240,7 @@ export class MirrorNodeClient {
         const queryParams = this.getQueryParams(queryParamObject);
         return this.request(`${MirrorNodeClient.GET_CONTRACT_RESULTS_ENDPOINT}${queryParams}`,
             MirrorNodeClient.GET_CONTRACT_RESULTS_ENDPOINT,
-            [400],
+            [400, 404],
```

Comment:
Please update mirrorNodeClient.spec.ts with this coverage

Reply:
Actually now i tested that 404 is returned only for no matches when querying block hash + index, for block number + index it will return 200 and empty array. So maybe the issue here is with the mirror node, not the relay?

Reply:
404's are applicable expected error codes.
For example asking for an account that doesn't exist will return a 404 and this is expected